exercises/all-your-base/all_your_base_test.go

```diff
@@ -0,0 +1,197 @@
+package allyourbase
+
+import "testing"
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
+	}
+}
```

Comment:
Would you mind moving this to the bottom of the file? I know we've been inconsistent with this, but I've realized that it's nice to keep it out of the way since it's more book-keeping and not essential to the problem at hand.